src/diffs/package-json.js

```diff
@@ -0,0 +1,37 @@
+// Heavily inspired by https://github.com/greenkeeperio/greenkeeper/blob/8be4bfc45dc991ec0c3331ac15b688728f5938a5/lib/diff-package-json.js
+
+const _ = require('lodash')
```

Comment:
Is lodash really necessary? I don't like bringing in lodash because there is a giant API and you end up only using a few functions. It's a huge pet-peeve of mine.